Extension/lib/filter/rules/url-filter-rule.js

```diff
@@ -311,6 +360,7 @@ UrlFilterRule.ignoreOptions = {
 };
 
 UrlFilterRule.contentTypes.DOCUMENT = UrlFilterRule.contentTypes.ELEMHIDE | UrlFilterRule.contentTypes.URLBLOCK | UrlFilterRule.contentTypes.JSINJECT;
+UrlFilterRule.contentTypes.DOCUMENT_LEVEL = UrlFilterRule.contentTypes.DOCUMENT | UrlFilterRule.contentTypes.GENERICHIDE | UrlFilterRule.contentTypes.GENERICBLOCK;

```

Comment:
I suggest renaming it to `DOCUMENT_LEVEL_EXCEPTIONS`

It should also contain:
`URLBLOCK`, `ELEMHIDE`, `JSINJECT`